Source/WebDriver/WebDriverService.h

```diff
@@ -32,13 +32,21 @@
 #include <wtf/JSONValues.h>
 #include <wtf/text/StringHash.h>
 
+#if ENABLE(WEBDRIVER_BIDI)
+#include "WebSocketServer.h"
+#endif
+
 namespace WebDriver {
 
 struct Capabilities;
 class CommandResult;
 class Session;
 
-class WebDriverService final : public HTTPRequestHandler {
+class WebDriverService final : public HTTPRequestHandler
+#if ENABLE(WEBDRIVER_BIDI)
+                            , public WebSocketMessageHandler
```

Comment:
Please fix the spacing here per style guide:
https://webkit.org/code-style-guidelines/#punctuation-member-init

Reply:
Fixed.